### DISCUSSION TITLE
autetication netlify with my gihub account

### DISCUSSION THREAD

**Discussion by Nasimrbi** [community/community#75494] [New to GitHub]
Hi, I want to log in with my GitHub account on Netlify and Vercel websites but both of them show me the error, on Netlify show me 'We already have a registered user with this email address. Log in to connect your GitHub account.', and on Vercel  show me 'Account not found.' 
and I don't know why !!! can you help me please

**Comment by Habibsultani** [comment]
Hi, 
If you receive a message saying there's a registered user with your email address, it's possible that you previously signed up with a different method (e.g., using an email/password instead of GitHub).
Try logging in using the email address and password associated with your Netlify account. If you've forgotten the password, use the "Forgot password" option to reset it.

For vercel: 
If you're receiving an "Account not found" error on Vercel, it might indicate that your GitHub account isn't associated with a Vercel account.
Try signing up again using your GitHub account. Make sure you are using the correct GitHub account associated with your Vercel account.

**Comment by mannubhai1** [comment]
My Netlify registration is not getting done.
That stupid website says my GitHub is suspicious, so it asked me for my identity proof.
I tried my passport, institute id card, aadhar card, but the fucking website can't even process them properly.
Cant register myself in netlify.

**Comment by ladunjexa** [comment]
Hi @NasimWeb and @mannubhai1,

If you haven't created deployments on Netlify or Vercel yet, consider completely deleting your accounts on those platforms. Additionally, visit the following link: https://github.com/settings/installations/. Look for `Vercel` and `Netlify, then click on `Configure`. In the danger zone, click on `Uninstall`.

After that:

**For Vercel**: Visit the following link: https://vercel.com/signup. For the free version, click on 'Hobby'. Then, choose your preferred username for your Vercel account and click on 'Continue with GitHub'.

**For Netlify**: Visit the following link: https://app.netlify.com/signup and click 'Sign Up with GitHub'. Follow the steps provided by the site.

If you're unsure how to delete your Netlify account, click here: [Netlify Deletion Guide Link](https://answers.netlify.com/t/deletion-of-netlify-account/106651). For Vercel, click here: [Vercel Deletion Guide Link](https://vercel.com/guides/how-do-i-delete-my-vercel-account).

Perform these steps from a browser that is already logged into your GitHub account.

If this response serves you well, please mark it as an answer :)
